.github/workflows/flake_finder.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Flake Finder
+
+on:
+  schedule:
+    - cron: "0 0/2 * * *"
```

Comment:
How about hourly runs but during off-peak hours? We'll get similar results without taking CI slots during work hours.

Reply:
I don't think we'll overwhelm the CI slots with this, it's only 4 jobs (smaller matrix than the operator). I'd definitely like to turn it down to two runs per day off-peak in a few days, after RC1.